dictionary.txt

```diff
@@ -1091,12 +1091,31 @@ Demon queen
 Non-binary
 UwU
 OwO
-Genderfae
+genderfae
 **Want kiss gorls, am lesbean
-Bi
-Plural
-Prog-rock
-Encrypted
-Pizza
-Kissable
-Huggabl
+pan
+plural
+prog-rock
+encrypted
+pizza
+kissable
+huggable
+meow
+kitten
+bunny
+wife
+wifeing
+queer
+gay
+evil
+**girls,,,,
+**smooching girls
+femboy
```

Comment:
I'm not sure about this one since femboy often means people identifying as he/him (well, i haven't really seen many people reclaim the term) and it seems sorta out of place for me because of that ?